infrastructure/unsigned/src/main/java/tech/pegasys/teku/infrastructure/unsigned/UInt64.java

```diff
@@ -37,6 +37,10 @@
    */
   static final long SQRT_MAX_VALUE = 4294967295L;
 
+  static final int SPECIAL_CASE_MULTIPLICAND = 64;
```

Comment:
more efficient to store as long and 64L?

Reply:
Probably neater to store as long but no difference to efficiency I don't think.

Reply:
Changed.